ui/components/CustomTableFooter.js

```diff
@@ -141,10 +103,9 @@ class CustomTableFooter extends Component {
 }
 
 CustomTableFooter.propTypes = {
-  // classes: PropTypes.object.isRequired,
   changePage: PropTypes.func.isRequired,
   rowsPerPage: PropTypes.number.isRequired,
   page: PropTypes.number.isRequired,
 };
 
-export default withStyles(defaultFooterStyles, { name: "CustomFooter" })(CustomTableFooter);
\ No newline at end of file
+export default withStyles(defaultFooterStyles, { name: 'CustomFooter' })(CustomTableFooter);
```

Comment:
It would be great if all the strings were either template literals(using quote:"backtick") or just normal strings with a single-quotation (using quote:"backtick") or quote:"double". (In this file we have all the 3 format of strings).